db/comdb2.c

```diff
@@ -2304,6 +2305,12 @@ static int read_lrl_option(struct dbenv *dbenv, char *line, void *p, int len)
         logmsg(LOGMSG_INFO,
                "Tables will be initialized with instant schema-change support\n");
         gbl_init_with_instant_sc = 1;
+    } else if (tokcmp(line, ltok, "dont_init_with_compr") == 0) {
```

Comment:
Why do we need this? Just add `init_with_compr none` to lrl or db which may require it.